spec/requests/users_controller_spec.rb

```diff
@@ -175,17 +175,11 @@
       context "with a trial user with no name set" do
         let(:user) { create(:user, :with_trial_role, name: nil) }
 
-        it "does not return error if name is not chosen and role is not changed" do
+        it "does not return error if name is not chosen" do
```

Comment:
What are your thoughts on renaming this? Seems the actual thing it's doing is redirecting if a name is missing

Reply:
I think the name still makes sense. It successfully updates the user if the name is not chosen, so long as there wasn't an existing name - rather than returning an error, which is what this is testing.

I guess it could be renamed to "successfully updates the user when a name is not set"

Reply:
We've renamed it to "successfully updates the user when a name is not set"